[DOCS] Update descs for custom expectations (#8923)
great-expectations/great_expectations: docs/docusaurus/docs/guides/expectations/custom_expectations_lp.md

</p>

<LinkCardGrid>

>   <LinkCard topIcon label="Create a Custom Column Aggregate Expectation" description="evaluates a single column and produces an aggregate Metric" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Create a Custom Column Aggregate Expectation" description="Evaluates a single column and produces an aggregate Metric" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_aggregate_expectations" icon="/img/custom_expectation_icon.svg" />
```

Author: capitalized all first letters in 0dc8a25de

>   <LinkCard topIcon label="Create a Custom Column Map Expectation" description="evaluates a single column and performs a yes or no query on every row in the column" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_map_expectations" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Create a Custom Column Map Expectation" description="Evaluates a single column and performs a yes or no query on every column row " href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

Author: capitalized all first letters in 0dc8a25de

>   <LinkCard topIcon label="Create a Custom Batch Expectation" description="evaluates an entire Batch, and answers a semantic question about the Batch" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Create a Custom Batch Expectation" description="Evaluates an entire Batch, and answers a semantic question about the Batch" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations" icon="/img/custom_expectation_icon.svg" />
```

Author: capitalized all first letters in 0dc8a25de

>   <LinkCard topIcon label="Create a Custom Column Pair Map Expectation" description="evaluates a pair of columns and performs a yes or no query about the row-wise relationship between the two columns" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_pair_map_expectations" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Create a Custom Column Pair Map Expectation" description="Evaluates a pair of columns and performs a yes or no query about the row-wise relationship between the columns" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_pair_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

Author: capitalized all first letters in 0dc8a25de

>   <LinkCard topIcon label="Create a Custom Multicolumn Map Expectation" description="evaluates a set of columns and performs a yes or no query about the row-wise relationship between the columns" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_multicolumn_map_expectations" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Create a Custom Multicolumn Map Expectation" description="Evaluates a set of columns and performs a yes or no query about the row-wise relationship between the columns" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_multicolumn_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

Author: capitalized all first letters in 0dc8a25de

>   <LinkCard topIcon label="Create a Custom Regex-Based Column Map Expectation" description="evaluates a single column and performs a yes or no regex-based query on every row in the column" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_regex_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Create a Custom Regex-Based Column Map Expectation" description="Evaluates a single column and performs a yes or no regex-based query on every column row" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_regex_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

Author: capitalized all first letters in 0dc8a25de

>   <LinkCard topIcon label="Create a Custom Set-Based Column Map Expectation" description="evaluates a single column and determines if each row in the column belongs to the specified set" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_set_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Create a Custom Set-Based Column Map Expectation" description="Evaluates a single column and determines if each row in the column belongs to the specified set" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_set_based_column_map_expectations" icon="/img/custom_expectation_icon.svg" />
```

Author: capitalized all first letters in 0dc8a25de

>   <LinkCard topIcon label="Create a Custom Query Expectation" description="runs Expectations against custom query results and makes intermediate queries to your database" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Create a Custom Query Expectation" description="Runs Expectations against custom query results and makes intermediate queries to your database" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_query_expectations" icon="/img/custom_expectation_icon.svg" />
```

Author: capitalized all first letters in 0dc8a25de

>   <LinkCard topIcon label="Create a Custom Parameterized Expectation" description="inherit from existing expectations to create a new customized Expectation" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_parameterized_expectations" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Create a Custom Parameterized Expectation" description="Inherits classes from existing Expectations and then creates a new customized Expectation" href="/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_parameterized_expectations" icon="/img/custom_expectation_icon.svg" />
```

Author: updated in 3eef32bef

>   <LinkCard topIcon label="Add auto-initializing framework support to a Custom Expectation" description="automate Expectation parameter estimation" href="/docs/guides/expectations/creating_custom_expectations/how_to_add_support_for_the_auto_initializing_framework_to_a_custom_expectation" icon="/img/custom_expectation_icon.svg" />

Reviewer: ```suggestion
  <LinkCard topIcon label="Add auto-initializing framework support to a Custom Expectation" description="Automates Expectation parameter estimation" href="/docs/guides/expectations/creating_custom_expectations/how_to_add_support_for_the_auto_initializing_framework_to_a_custom_expectation" icon="/img/custom_expectation_icon.svg" />
```
